pyworklistserver/__main__.py

```diff
@@ -133,9 +133,9 @@ def _handle_signal(self, sig, frame):
 
 
 if __name__ == '__main__':
-    server_config, logfile, seedfile, reproduce, configfile = _get_serverconfig_from_commandline()
+    server_config, logfile, seedfile, reproduce, worklist_config = _get_serverconfig_from_commandline()
     logger = _configure_logger(logfile, logging.DEBUG if server_config.verbose else logging.WARN)
-    inserted_config = _get_userconfig_from_commandline(configfile, logger)
+    worklist_config = _get_userconfig_from_commandline(worklist_config)

```

Comment:
Here worklist_config is changing meaning from being a file reference to becoming a dict. Maybe it is better to rename the worklist_config input argument to worklist_config_file. This way, we don't use the same name for different concepts.